### PR TITLE
Creates Holosign Creator for CE

### PR DESCRIPTION
**Pull request by JamieD1**
Added Holo barrier tool for the CE,

Can change between engineering and atmos barriers,

Will add to locker if duffel bag gets approved if not, will afterwards.

#### Changelog

:cl:  
rscadd: Added CE Holo Barrier Tool
/:cl:

### CODE REVIEW

**Comment by yacabo** [comment]
Can it fit in a toolbelt?

**Comment by JamieD1** [comment]
> Can it fit in a toolbelt?

Now it can!

Spawns in CE's by default

**Comment by athena413** [comment]
yes

**Comment by JamieD1** [comment]
@alexkar598 Please review

File: code/game/objects/items/holosign_creator.dm [hunk 1]
```diff
@@ -13,6 +13,7 @@
 	throw_range = 7
 	item_flags = NOBLUDGEON
 	var/list/signs = list()
+	var/list/holodesigns = list()
```
Comment (alexkar598) [inline code/game/objects/items/holosign_creator.dm]: Move this to the multi subtype

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/holosign_creator.dm (hunk 1 above)
Comment (Hopekz) [reply code/game/objects/items/holosign_creator.dm]: Looks like this has been modified. Doing a final check before the merge

**Pull request merged**